docs/runbooks/extending_workspace_time/extending_workspace.md

```diff
@@ -35,3 +35,20 @@ Following the instructions for editing the workspace Time To Live will put this
 
 This will not work if there is no record in place.
 At this point it's recommended to create the PR environment through the normal CI Process of bumping the PR via a code change or merge.
+
+## removing workspace
+When UAT is completed, it might be a good idea to remove the environment so that we're being more efficient with the cost.
+ The simplest way to remove an environment is to:
+
+1. locate the record as per steps 3 to 7 for changing Time to Live
+2. Select the workspace required for deletion
+3. click on `Actions` dropdown and select `Delete items`
+4. At the confirmation click `Delete`
+5. The cleanup job will destroy the environment up on the next clean up run.
+
+However, should you need to do an immediate destruction of the workspace, you can do this from your local shell by:
```

Comment:
Could the word 'by' be deleted on the end? It makes the list read oddly by being in the wrong tense.

Reply:
noted
```suggestion
However, should you need to do an immediate destruction of the workspace, you can do this from your local shell:
```